test-integration/test_integration/test_redis_queue.py

```diff
@@ -421,6 +432,16 @@ def test_queue_worker_error_after_output(docker_network, docker_image, redis_cli
             "logs": ["a printed log message"],
         }
 
+        response = json.loads(redis_client.blpop("response-queue", timeout=10)[1])
+        assert response == {
+            "x-experimental-timestamps": {
+                "started_at": mock.ANY,
+            },
+            "status": "processing",
+            "output": ["hello bar"],
+            "logs": mock.ANY,  # includes a stack trace
```

Comment:
The surrounding responses have assertions for the stack trace but this one doesn't. Is that intentional?

Reply:
It didn't seem worth doing that assertion twice, but I've moved it to the earliest logs it should appear in for both tests 👍